vscripts/client/cl_fullscreen_map.nut

```diff
@@ -66,6 +82,9 @@ function ShowFullMap( player )
 
 function ResetAndShowMinimap( player )
 {
+	if ( !level.showingFullscreenMap )
+		return
```

Comment:
This cannot be put here, because `ResetAndShowMinimap` is also used externally here:
https://github.com/p0358/bme_scripts/blob/981ab15008c4e4c15db0ebdc98af6dbf05b0617b/vscripts/client/cl_player.nut#L3474

If this was put there, I think it'd make a bug where player dying would make the fullscreen map linger

Reply:
That doesn't seem to happen in my testing ( first death was hold, the second one was toggle ). The reason i added these is because the scoreboard script also uses something similar: https://github.com/NachosChipeados/bme_scripts/blob/d29afb8ec686315e3aa581fd3df6db4ad2cc8f42/vscripts/client/cl_scoreboard.nut#L386 and https://github.com/NachosChipeados/bme_scripts/blob/d29afb8ec686315e3aa581fd3df6db4ad2cc8f42/vscripts/client/cl_scoreboard.nut#L880. I can still remove them if you'd like though

https://github.com/user-attachments/assets/b291a888-486a-4fdb-972b-0e073646431d

Reply:
True, I think scoreboard is mainly because it'd start a thread for refreshing it otherwise, so there they very much wouldn't want that to happen twice.

Actually I think I was too sleepy when looking at it, it calls the func and it'll dismount the fullscreen map if it's shown, it'd only not run if it's not shown, and technically nothing else should flip the boolean outside of these funcs, so in theory it might be fine, but then again it's still technically somewhat unneccessary, so I think I'd still lean towards removing those

Reply:
Alright